src/Constructorio_NET.Tests/client/modules/AutocompleteTests.cs

```diff
@@ -83,11 +83,11 @@ public async Task GetAutocompleteResultsShouldReturnResultWithVariationsMap()
             );
 
             Assert.NotNull(res.ResultId, "Result id exists");
-            Assert.AreEqual(
-                JObject.Parse(reqVariationsMap.ToString()),
-                variationMapResult,
-                "Variations Map was passed as parameter"
-            );
+            var actualVariationMap = JObject.Parse(reqVariationsMap.ToString());
+
+            Assert.IsTrue(actualVariationMap.ContainsKey("filter_by"), "Should contain filter_by");
+            Assert.IsTrue(actualVariationMap.ContainsKey("group_by"), "Should contain group_by");
+            Assert.IsTrue(actualVariationMap.ContainsKey("values"), "Should contain values");
```

Comment:
I'm not against the check being more loose here, but we should remove line 81-83 since that variable isn't in use anymore. 

This also means that the test won't detect if the variations_map configured in the SDK is actually applied in the API response. This is usually fine if we already check the final network request being made, but we don't do that atm. Mind adding a couple of assert statements checking the some of the values as well?

In the future, maybe we can ensure that JObjects returned in the response can be re-serialized into the model like how we're doing for [PFEs](https://github.com/Constructor-io/constructorio-netcore/blob/main/src/Constructorio_NET.Tests/client/modules/SearchTests.cs#L167-L170). That way we wouldn't have to maintain a blob of JSONString for our tests